Makefile

```diff
@@ -123,6 +148,93 @@ check.lint: ; $(info $(H) running golint on packages...)
 build: setup
 	bazel $(BAZEL_STARTUP_ARGS) build $(BAZEL_BUILD_ARGS) //...
 
+#-----------------------------------------------------------------------------
+# Target: go build
+#-----------------------------------------------------------------------------
+
+.PHONY: go-build
+
+.PHONY: pilot
+pilot: vendor
+	go install istio.io/istio/pilot/cmd/pilot-discovery
+
+.PHONY: pilot-agent
+pilot-agent: vendor
+	go install istio.io/istio/pilot/cmd/pilot-agent
+
+.PHONY: istioctl
+istioctl: vendor
+	go install istio.io/istio/pilot/cmd/istioctl
+
+.PHONY: sidecar-initializer
+sidecar-initializer: vendor
+	go install istio.io/istio/pilot/cmd/sidecar-initializer
+
+.PHONY: mixs
+mixs: vendor
+	go install istio.io/istio/mixer/cmd/mixs
+
+.PHONY: mixc
+mixc: vendor
+	go install istio.io/istio/mixer/cmd/mixs
+
+go-build: pilot istioctl pilot-agent sidecar-initializer mixs mixc
```

Comment:
I wonder if it should depend also on `depend.vendor` or similar. Or it's up to the user to deal with dependencies? Current build (with bazel) does that automatically, correct?